aiida_mlip/calculations/md.py

```diff
@@ -0,0 +1,128 @@
+"""Class to run md calculations."""
+
+from aiida.common import datastructures
+import aiida.common.folders
+from aiida.engine import CalcJobProcessSpec
+import aiida.engine.processes
+from aiida.orm import Dict, SinglefileData, Str, StructureData, TrajectoryData
+
+from aiida_mlip.calculations.base import BaseJanus
+
+
+class MD(BaseJanus):  # numpydoc ignore=PR01
+    """
+    Calcjob implementation to run geometry MD calculations using mlips.
+
+    Methods
+    -------
+    define(spec: CalcJobProcessSpec) -> None:
+        Define the process specification, its inputs, outputs and exit codes.
+    prepare_for_submission(folder: Folder) -> CalcInfo:
+        Create the input files for the `CalcJob`.
+    """
+
+    DEFAULT_TRAJ_FILE = "aiida-traj.xyz"
+    DEFAULT_STATS_FILE = "aiida-stats.dat"
+    DEFAULT_SUMMARY_FILE = "md_summary.yml"
```

Comment:
I had them private but then actually I need to use them in the parser so I removed the `_`
But because I didn't use all of them in the parser those that I wasn not using I left them private, which I think is not nice and consistent, should I just change them all to not private?

Reply:
Letting a power-user see and overwrite these is sensible. Not private is probably best.